app/admin/roomdetails/page.tsx

```diff
@@ -0,0 +1,161 @@
+"use client";
+import HeadingBook from "@/components/roomDetails/HeadingBook"; 
+import Details from "@/components/roomDetails/Details";
+import Category from "@/components/roomDetails/Category";
+import ServiceAd from "@/components/roomDetails/ServiceAd";
+import Additional from "@/components/roomDetails/Additional";
+import Gallery from "@/components/roomDetails/Gallery";
+import { Grid, Box, Container, Card, Typography } from "@mui/material";
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Button from "@mui/material/Button";
+import AddAPhotoIcon from "@mui/icons-material/AddAPhoto";
+
+const VisuallyHiddenInput = styled("input")({
+  clip: "rect(0 0 0 0)",
+  clipPath: "inset(50%)",
+  height: 1,
+  overflow: "hidden",
+  position: "absolute",
+  bottom: 0,
+  left: 0,
+  whiteSpace: "nowrap",
+  width: 1,
+});
+const roomDetails = () => {
+  return (
+    <>
+      <Box sx={{ backgroundColor: "", padding: "10px 0", width: "100%" }}>
+        <Container>
+          <HeadingBook />
+        </Container>
+      </Box>
+      <Box
+        sx={{
+          display: "grid",
+        }}
+      >
+        {/* (1) The Details component is imported and rendered here. */}
+        <Container>
+          <Grid container spacing={2}>
+            {/* <RoomDetailsForm /> */}
+            <Grid item xs={12} sm={12} md={6} lg={6} xl={6}>
+              <Details />
+            </Grid>
+            {/* (2) The Gallery component is imported and rendered here. */}
+            <Grid item xs={12} sm={12} md={6} lg={6} xl={6}>
+              <Typography
+                style={{
+                  color: "Gray",
+                  fontSize: "16px",
+                  lineHeight: "120%",
+                  fontStyle: "normal",
+                  letterSpacing: "0.12px",
+                  fontWeight: "700",
+                  marginTop: "10px",
+                }}
+              >
+                Cover Image
+              </Typography>
+              <Card
+                elevation={0}
+                sx={{
+                  height: "299px",
+                  Width: "auto",
+                  border: "1px solid",
+                  borderRadius: "8px",
+                  borderColor: "#D0D5DD",
+                  flexShrink: "0",
+                  paddingTop: "25%",
```

Comment:
@wickramasinghe07 can you use MUI spacing for padding and margin without using %?